### DISCUSSION TITLE
Nên có phiên bản Fcitx5-Bamboo

### DISCUSSION THREAD

**Discussion by tuandungb** [BambooEngine/ibus-bamboo#488] [Ideas]
Xin chào, Mình là một người dùng thông thường. Hiện đang dùng Opensuse TW KDE. Với KDE hiện nay thì cài ibus thực sự khá vất vả và ko ổn định, thân thiện như với Fcitx5. Với các version bộ gõ như unikey, bogo .... hiện nay thì gần như là lâu lắm rồi ko thấy có cập nhật gì mới. Mình thấy bamboo cập nhật rất thường xuyên và thấy có tính năng gõ mà ko gạch chân. Nên mình rất muốn bamboo phát hành phiên bản cho Fcitx5, song song với ibus. Như thế sẽ rất tiện và giải quyết đc bài toán cơ bản nhất hiện nay khi người dùng loay hoay với bộ gõ nào giống unikey trên win. 
Thân

**Comment by goatastronaut0212** [comment]
Chẳng phải là cũng có 1 phiên bản khác của fcitx5-bamboo rồi sao? Mình nghĩ nên đóng góp cho dự án đó thay vì phát minh lại bánh xe, mình không biết là dự án đó có gì khác so với Ibus nhưng vì nó sẽ khá tốn thời gian trừ khi bạn muốn làm việc đó. Mình nghĩ ông dev bên đó khá là thân thiện trên telegram cũng sẽ dễ đón nhận PR nhưng mà bên đó có vẻ không thích unofficial repo cho lắm trừ khi bạn muốn package nó lên official repo của các distro để người dùng dễ cài đặt lấy gói từ đó hơn.

Vấn đề của ibus có khá nhiều nhưng tất nhiên Ibus cũng sẵn sàng đón nhận các PR của bạn mà. Mình cũng không rõ là người dùng cuối có vấn đề gì với Ibus nữa? Bạn mô tả rõ hơn được không? Mình thấy người dùng của Ibus-bamboo họ marketing dự án này khá là nhiều nếu lướt trên các mạng xã hội thì đa phần đều chỉ về dự án này hết.
